scripts/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "ES5",
-    "module": "commonjs",
```

Comment:
I'm not sure we can change this without a major version?

Reply:
This is for the scripts directory! Nothing here actually gets built or packaged, it just stops the language server from complaining.